assets/js/components/surveys/SurveyShow.jsx

```diff
@@ -70,12 +71,15 @@ class SurveyShow extends Component<any, State> {
       contacted: false,
       uncontacted: false,
       stopUnderstood: false,
+      paused: false,
     }
   }
 
   componentWillMount() {
     const { dispatch, projectId, surveyId } = this.props
-    dispatch(actions.fetchSurveyIfNeeded(projectId, surveyId))
+    dispatch(actions.fetchSurveyIfNeeded(projectId, surveyId)).then((response) => {
+      this.setState({ paused: response.state == "paused" })
```

Comment:
Does it make sense to have a function that checks the survey's state instead of adding a new variable to the state? So you don't have to maintain it separately.